lib/create-http-client.ts

```diff
@@ -1,5 +1,7 @@
 import copy from 'fast-copy'
 import qs from 'qs'
+import type { AxiosStatic } from 'axios'
```

Comment:
I'd recommend to hide all the Axios* names/types behind re-exported ones to make potential switches easier.

Reply:
Not sure if we need to do it now, realistically if we mostly likely switch from `axios` to `redaxios` and have exactly the same API

Reply:
Do they export the same types?